### DISCUSSION TITLE
在chrome83.04中使用NaiveUi2.38.1以上版本，会有诸多样式问题，请问如何解决

### DISCUSSION THREAD

**Discussion by Augustjoe** [tusen-ai/naive-ui#6126] [Q&A]
例如使用DataTable组件，无法正确展示滚动条

**Comment by Sepush** [comment]
当前 Chrome 正式版版本号已经来到 127.x 83 已经是4年前的chrome版本了 我想我们不会特别花精力兼容

**Comment by jahnli** [comment]
This issue does not have any recent activity. If you are still experiencing similar problems, open a new error, including a minimal copy of the problem
